src/app/Components/calendar-view/calendar-view.component.ts

```diff
@@ -37,14 +41,17 @@ export class CalendarViewComponent implements OnInit {
   }
 
   getWeekdayForFirstOfMonth(month: number) {
-    const day = new Date(this.year + '-' + month + '-01').getDay();
+    let day;
+
+    if (month <= this.month.id) {
+      day = new Date(this.year + '-' + month + '-01').getDay();
+    } else {
+      day = new Date(this.year + '-' + month + '-01').getDay() + 1;
+    }
+
```

Comment:
what is this part for?

Reply:
this is the only way I could get the days to match up for any month after the current month. It was consistently off by one week day